Form-Controls/index.html

```diff
@@ -14,14 +14,50 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
+    <div>
+      <lable for="name">Name</lable>
```

Comment:
Double-check the spelling for this element here (and other cases later on the form) - you might notice that the way that the labels appear in the form is slightly different depending on the spelling used, and this is because if a web browser doesn't recognise the name of the element being used (in this case "lable" rather than "label") it will just treat it as if it were like an inline HTML element like `<span>`. You'll also notice that you aren't able to select fields by clicking the label where "lable" is used, but can where "label" is used. (I realise English spelling rules are annoyingly inconsistent!!)